src/main/resources/com/google/api/codegen/py/types.snip

```diff
@@ -0,0 +1,47 @@
+@extends "py/common.snip"
+
+@snippet generate(view)
+  {@header(view.fileHeader)}
+
+  {@body(view)}
+@end
+
+@private header(fileHeader)
+    {@licenseSection(fileHeader)}
+
+    {@renderImportSection(fileHeader.importSection)}
+@end
+
+@private licenseSection(fileHeader)
+    @join line : fileHeader.copyrightLines
+        @# {@line}
+    @end
+    {@BREAK}@#
+    @join line : fileHeader.licenseLines
+        @# {@line}
+    @end
+@end
+
+@private body(view)
+  names = []
+  for module in ({@importedModules(view.fileHeader.importSection.appImports)}):
```

Comment:
Will this have a trailing comma if there is only one imported module? There either needs to be a trailing comma in that case, or this needs to become a list rather than a tuple. (I expect the latter is easier to do.)